src/init/features/apphosting/index.ts

```diff
@@ -212,9 +213,32 @@ export async function onboardRollout(
   const buildId = await apphosting.getNextRolloutId(projectId, location, backendId, 1);
   const buildOp = await apphosting.createBuild(projectId, location, backendId, buildId, buildInput);
 
-  const rolloutOp = await apphosting.createRollout(projectId, location, backendId, buildId, {
+  const rolloutBody = {
     build: `projects/${projectId}/locations/${location}/backends/${backendId}/builds/${buildId}`,
-  });
+  };
+
+  let tries = 0;
+  let done = false;
+  while (!done) {
+    tries++;
+    try {
+      await apphosting.createRollout(projectId, location, backendId, buildId, rolloutBody, true);
+      done = true;
+    } catch (err: unknown) {
```

Comment:
can we be more specific with the error code here?

Reply:
I don't know what you mean.

`unknown` is allowed and expected - you can't be more specific else you get an error  like this: `Catch clause variable type annotation must be 'any' or 'unknown' if specified.`

Can I check to see if it's an expected error for this scenario? Sure.